src/sentinel.c

```diff
@@ -2055,6 +2056,20 @@ void rewriteConfigSentinelOption(struct rewriteConfigState *state) {
     rewriteConfigRewriteLine(state,"sentinel announce-hostnames",line,
                              sentinel.announce_hostnames != SENTINEL_DEFAULT_ANNOUNCE_HOSTNAMES);
 
+    /* detirmine if the config uses the legacy known-slave or known-replica term */
+    const char *knownReplicaConfig;
+
```

Comment:
maybe we do not need this space

Reply:
Is there any utility to auto fix format?